### PR TITLE
[-] Templates - Replace unconventional `bin/www` script with `server.js`

### PR DESCRIPTION
**Pull request by rap2hpoutre**

### CODE REVIEW

File: services/dumper.js [hunk 1]
```diff
@@ -64,7 +64,6 @@ function Dumper(config) {
       name: config.appName.replace(/ /g, '_').toLowerCase(),
       version: '0.0.1',
       private: true,
-      scripts: { start: 'node ./bin/www' },
```
Comment (VincentMolinie) [inline services/dumper.js]: [Spec] We cannot remove this as it will break the start with nodemon

Comment (rap2hpoutre) [reply services/dumper.js]: I used to run `nodemon bin/www` and now I have to run `nodemon server.js`. Was there another (better) way?

Comment (VincentMolinie) [reply services/dumper.js]: `nodemon` simply. `Nodemon` use the start by default ^^

Comment (rap2hpoutre) [reply services/dumper.js]: OK `nodemon` worked too.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,8 @@
 # Change Log
 
 ## [Unreleased]
+### Changed
+- Templates - Relace unconventional `bin/www` script with `server.txt`.
```
Comment (VincentMolinie) [inline CHANGELOG.md]:
```suggestion
- Templates - Replace unconventional `bin/www` script with `server.txt`.
```

**Pull request merged**